content/docs/next-release/spm.md

```diff
@@ -9,6 +9,8 @@ as experimental to signal that this feature will be in flux while bugs are fixed
 enhancements are added in response to community feedback.
 {{< /info >}}
 
+![Service Performance Monitoring](/img/frontend-ui/spm.png)
```

Comment:
I think we should also add this to the intro page

Reply:
added in https://github.com/jaegertracing/documentation/commit/83703bdc5b7e371578e3b46cafb70f3e162d2fa9